src/lib/SapphireClient.ts

```diff
@@ -1,6 +1,7 @@
 import type { Piece, Store } from '@sapphire/pieces';
 import { Client, ClientOptions, Message } from 'discord.js';
 import { join } from 'path';
+import './extensions/SapphireMessage';
```

Comment:
Could extensions be put in its own spot/block?
Possibly below all the other imports.

Reply:
VSCode organize imports is going to undo this btw... @kyranet as well since you already committed the change.